tox.ini

```diff
@@ -52,9 +52,8 @@ lines_after_imports=2
 include_trailing_comma=1
 multi_line_output=3
 skip=.tox,migrations
-not_skip=__init__.py
 use_parentheses=1
 known_django=django
-known_future_library=future,six
+known_future_library=future
```

Comment:
@willbarton Can this be removed entirely at this point?

Reply:
@Scotchester Agreed that this should be removed from isort configuration and I verified that tox continues to run without errors.